src/include/simeng/models/emulation/Core.hh

```diff
@@ -9,6 +9,8 @@
 #include "simeng/MemoryInterface.hh"
 #include "simeng/RegisterFileSet.hh"
 #include "simeng/arch/Architecture.hh"
+#include "simeng/arch/aarch64/ExceptionHandler.hh"
```

Comment:
For each of the 3 core models, could you add a `coreID_` member variable which defaults to 0, and has a `// TODO` above it stating we need to implement functionality for assigning unique IDs when we go multicore.

Then also have this passed correctly to the syscall handler when appropriate

Reply:
Added coreId_ member variable with getter and use in exceptionHandler core_.sendSyscall call